model/user.go

```diff
@@ -1,27 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Models will be completely removed and we will use those defined on ernest-go-sdk instead.